src/Command/CleanUpCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @file
+ * Command to clean up local stored images after upload detected.
+ */
+
+namespace App\Command;
+
+use App\Entity\Cover;
+use App\Repository\CoverRepository;
+use App\Service\CoverStoreService;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CleanUpCommand.
+ */
+class CleanUpCommand extends Command
+{
+    private $coverRepository;
+    private $coverStoreService;
+    private $entityManager;
+
+    protected static $defaultName = 'app:image:cleanup';
+
+    /**
+     * CleanUpCommand constructor.
+     */
+    public function __construct(CoverRepository $coverRepository, CoverStoreService $coverStoreService, EntityManagerInterface $entityManager)
```

Comment:
Consider adding an ignore rule to phpcs regarding line length.

Reply:
https://github.com/danskernesdigitalebibliotek/ddb-cover-service/blob/develop/phpcs.xml.dist#L18

Reply:
Added :-)